src/Value/Value.php

```diff
@@ -70,20 +70,26 @@ public static function parseValue(ParserState $oParserState, array $aListDelimit
             if (count($aStack) === 1) {
                 return $aStack[0];
             }
-            $iStartPosition = null;
-            while (($iStartPosition = array_search($sDelimiter, $aStack, true)) !== false) {
+            $aNewStack = [];
+            for ($iStartPosition = 0; $iStartPosition < count($aStack); ++$iStartPosition) {
```

Comment:
`count` is relatively expensive. We should move it out of the loop and store the number of elements in a local variable to avoid calling `count` on each loop iteration.

Reply:
Good point, fixed.

Reply:
> `count` is relatively expensive.

[It is actually O(1)](https://stackoverflow.com/questions/5835241/is-phps-count-function-o1-or-on-for-arrays/5835419#5835419), but noneless [slightly quicker with the result cached in a local variable (maybe 50µs per reference in 2014](https://stackoverflow.com/a/23410780/2511031).